src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
+    <header class="header-container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Company logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li>
+            <a
+              href="/apple"
+              class="nav-item is-active"
```

Comment:
You're not using the BEM methodology correctly here. The way to go would be, since you have `class="navigation"` on your block `nav`, it would be better to have `class="navigation__list` in your `<ul>`, for example, and `nav__item` in the `<a>`